lightkurve/mast.py

```diff
@@ -61,86 +69,242 @@ def search_kepler_products(target):
-    return products[mask]
+    if verbose:
+        astropylog.setLevel('INFO')
+    else:
+        astropylog.setLevel('ERROR')
+
+    #If we are asking for all cadences (long and short) and didn't specify a month, override it.
+    if (cadence in ['any', 'both']) & (month is None):
+        month = np.arange(3) + 1
+
+    #anys and all should be a list of numbers
+    if month in ['all','any']:
+        month = np.arange(3) + 1
+    if campaign in ['all','any']:
+        campaign = np.arange(0, 22)
+    if quarter in ['all','any']:
+        quarter = np.arange(0, 18)
+    if month is not None:
+        if not hasattr(month,'__iter__'):
+            month = [month]
+
+    #Grab the products
+    products = search_kepler_products(target=target, filetype=filetype, cadence=cadence,
+                                      quarter=quarter, campaign=campaign,
+                                      radius=radius)
+    if len(products) < 1:
+        raise ArchiveError("No {} File found for {} at MAST.".format(filetype, target))
+
+
+    #Limit to the correct number of hits based on ID. If there are multiple versions
+    #of the same ID, this shouldn't count towards the limit.
+    if targetlimit is not None:
+        ids = np.asarray([p.split('/')[-1].split('-')[0].split('_')[0][4:]
+                          for p in products['dataURI']], dtype=int)
+        if len(np.unique(ids)) < targetlimit:
+            logging.warning('Target return limit set to {} '
+                            'but only {} unique targets found. '
+                            'Trying increasing search radius. '
+                            '(radius currently set to {} arcseconds)'
+                            ''.format(targetlimit, len(np.unique(ids)), radius))
+        okids = ids[np.sort(np.unique(ids, return_index=True)[1])[0:targetlimit]]
+        mask = np.zeros(len(ids), dtype=bool)
+        idsort = np.zeros(len(ids))
+        for idx, o in enumerate(okids):
+            p = ids == o
+            idsort[p] = int(idx)
+            mask |= p
+        products['ids'] = idsort
+        products = products[mask]
+
+    #For Kepler short cadence data there are additional rules, so find anywhere
+    #where there is short cadence data...
+    scmask = np.asarray(['Short' in d for d in products['description']]) &\
+             np.asarray(['kplr' in d for d in products['dataURI']])
+    if np.any(scmask):
+        #Error check the user if there's multiple months and they didn't ask
+        #for a specific one
+        if month is None:
+            raise ArchiveError("Found {} different Target Pixel Files "
+                               "for target {} in Quarter {}. "
+                               "Please specify the month (1, 2, or 3)."
+                               "".format(len(products), target, quarter))
+        #Get the short cadence date lookup table.
+        table = ascii.read('/Users/ch/K2/repos/lightkurve/lightkurve/data/short_cadence_month_lookup.csv')
```

Comment:
Change this to `os.path.join(PACKAGEDIR, 'data', 'short_cadence_month_lookup.csv')`